account_product_fiscal_classification/models/wizard_change_fiscal_classification.py

```diff
@@ -27,11 +27,14 @@ def _default_old_fiscal_classification_id(self):
         required=True, domain="[('id', '!=', old_fiscal_classification_id)]")
 
     # View Section
-    @api.one
```

Comment:
@rvalyi not required but can be placed. It will not have any impact in the result because old api is not supported in V10, also will not raise any issue because we can loop through a single recordset as well. I would like to know if it will make an impact ?